dev/acf-export.json

```diff
@@ -901,7 +729,7 @@
                 "label": "Related Social",
                 "name": "cluster_related_tweets",
                 "type": "repeater",
-                "instructions": "Add the URLs to relevant tweet or facebook posts.",
+                "instructions": "Add the URLs to relevant tweet or facebook posts",
```

Comment:
ah, it looks like all of the small bean corrections from yesterday's PR got removed :)